pyproject.toml

```diff
@@ -8,6 +8,13 @@ readme = "README.md"
 repository = "https://github.com/anancarv/python-artifactory"
 documentation = "https://github.com/anancarv/python-artifactory"
 keywords = ["artifactory"]
+classifiers = [
+  "Development Status :: 4 - Beta",
```

Comment:
In my opinion, we can set this field to Production/Stable because I'm not sure we will do the `.save()` now. However, the package is production usable. If we add some features in 2 months, they can be assimilated to minor releases